js/main.js

```diff
@@ -23,6 +32,182 @@ function addCats(eachCat) {
   catName.className = 'cat-name';
   catImg.className = 'cat-img';
   catName.textContent = eachCat.name;
+  col.dataset.view = eachCat.name;
   catImg.setAttribute('src', eachCat.image.url);
   return col;
 }
+
+function addCatInfoView() {
```

Comment:
This should be `renderCatDetailView`. You should also be passing in the cat data as an argument, and using that data to populate this structure.